src/SilverorangePEAR/ruleset.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<ruleset name="SilverorangePEAR">
+  <description>Silverorange PEAR coding standard for packages missing documentation</description>
+
+  <!--
+    Import entire PEAR standard.
+  -->
+  <rule ref="PEAR" />
+
+  <!--
+    Ignore rules concerning doc comments
+  -->
+  <rule ref="Generic.Commenting.DocComment">
```

Comment:
This syntax in phpcs.xml is used to suppress specific messages within a sniff class. For example, if we wanted to require class comments but ignore missing `@category` tags we'd use this syntax.

In this case we want to exclude most commenting requirements so we should use the `<exclude>` element inside the `<rule>`

We do enforce class comments but not category and author tags so we could use:

```xml
<rule ref="PEAR.Commenting.ClassComment.MissingCategoryTag">
    <severity>0</severity>
  </rule>
  <rule ref="PEAR.Commenting.ClassComment.MissingAuthorTag">
    <severity>0</severity>
  </rule>
  <rule ref="PEAR.Commenting.ClassComment.MissingLinkTag">
    <severity>0</severity>
  </rule>
```